src/Common/nunit/ConsoleOptions.cs

```diff
@@ -382,8 +382,10 @@ private void ConfigureOptions()
             this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
                 v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
 
+#if !NETCF
```

Comment:
Actually, #if !NETCF is exactly how we have done all these conditions. This is a file that is shared between the console runner and NUnitLite. Since the console runner is only built for desktop .NET, NETCF always implies NUNITLITE. We could test for both, but that's not how we have done it elsewhere. Of course, if we decided to build a full console runner for CF, we would end up changing many conditions.